metric-anomaly-data-model/src/main/proto/org/hypertrace/alert/engine/eventcondition/config/service/v1/metric_anomaly_event_condition.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package org.hypertrace.alert.engine.eventcondition.config.service.v1;
+
+message MetricAnomalyEventCondition {
+  MetricAnomalyEvaluationRule evaluation_rule = 1;
+  NotificationCondition notification_condition = 2;
+}
+
+message MetricAnomalyEvaluationRule {
+  MetricSelection metric_selection = 1;
+  repeated ViolationCondition violation_condition = 2;
```

Comment:
we have renamed the `baseline_condtion` to `violation_condtion` as it was getting confused with baseline calculation while talking with @rish691 on this.